src/main/java/me/jellysquid/mods/lithium/common/chunk/SlimeChunkStorage.java

```diff
@@ -0,0 +1,37 @@
+package me.jellysquid.mods.lithium.common.chunk;
+
+import java.lang.Thread;
+import java.util.HashSet;
+
+import net.minecraft.server.world.ServerWorld;
+import net.minecraft.util.math.ChunkPos;
+ 
+public class SlimeChunkStorage{
```

Comment:
Might be better to store this as a field on the Chunk rather

Reply:
That was the initial idea but the slime spawning doesnt get sent a chunk so I cant really do anything about that because there is no good way of getting the chunk in ram from the blockPos because it would mean i have to deserialize it again each spawn

Reply:
Spawning would only happen in loaded chunks, though. It should be fine to get the chunk from the world

Reply:
Okay yea after some work this is completely gone in favor of your idea. (It doesnt work but still thats how its gonna be done)